views/api.pug

```diff
@@ -0,0 +1,133 @@
+extends layout.pug
+
+block content
+    .row
+        .col-12
+
+            h2.text-center.mb-4 API
+
+            p The BootstrapCDN API allows you to:
+
+            ul
+                li
+                    a(href='#list-package-versions') list package versions
+                li
+                    a(href='#list-package-files') list package files
+
+            p The API is free to use and imposes no rate limits (however, if you plan to make 100+ RPM for longer periods of time, you should contact us first).
+
+            p Looking for a search endpoint? You can use&nbsp;
+                a(href='https://github.com/npm/registry/blob/master/docs/REGISTRY-API.md#get-v1search') the official npm API
+                |
+                | or
+                |
+                a(href='https://github.com/algolia/npm-search') Algolia’s npm search
+                |.
+
+            h3
+                strong Let us know how you use this API
+
+            p
+                strong If you create a tool/plugin/etc. which uses this API, please include a link to your tool in the
+                    |
+                    |
+                    code User-Agent
+                    |
+                    | header so that we can learn more about how this API is being used.
+
+            h3 Endpoints
+
+            p
+                code #{config.siteurl}/api/v1
+
+            ul
+                li All responses are in JSON.
+                li All error responses have
+                    |
+                    |
+                    code status
+                    |
+                    | and
+                    |
+                    code message
+                    |
+                    | properties.
+
+            h4#list-package-versions
+                code /:name
+                |
+                | - List package versions
+
+            ul
+                li
+                    code:name
+                    |
+                    | - package name
+                    sup &nbsp;
+                        a(href='#notes') 1
+
+            p
+                strong Example
+
+            p GET &nbsp;
+                a(href='/api/v1/bootstrap' target='_blank') #{config.siteurl}/api/v1/bootstrap
+
+            pre.hljs
```

Comment:
We don't use highlight.js.

Reply:
Sure. This was primarily just a byproduct of auto-converting from markdown to pug based on https://github.com/jsdelivr/data.jsdelivr.com/blob/master/README.md. I can remove the unnecessary class.